app/views/organizations/_side_menu.html.erb

```diff
@@ -1,7 +1,10 @@
 <nav class="menu">
   <span class="menu-heading">Classroom settings</span>
-  <%= link_to 'Classroom profile', edit_organization_path, class: "menu-item #{'selected' if current_page?(edit_organization_path)}" %>
+  <%= link_to 'Classroom profile', edit_organization_path(@organization), class: "menu-item #{'selected' if current_page?(edit_organization_path(@organization))}" %>
```

Comment:
What about extracting menu items into another template?

in **organizations/_side_menu.html.erb**:

```
<%= render partial: 'organizations/menu_item', locals: { label: 'Classroom profile', path: edit_organization_path(@organization) } %>
```

in **organizations/_menu_item.html.erb**:

```
<%= link_to label, path, class: "menu-item #{'selected' if current_page?(path)}" %>
```